realscale.inx

```diff
@@ -2,19 +2,91 @@
 <inkscape-extension xmlns="http://www.inkscape.org/namespace/inkscape/extension">
   <_name>RealScale</_name>
   <id>org.inkscape.effect.realscale</id>
+  
   <dependency type="executable" location="extensions">cubicsuperpath.py</dependency>
   <dependency type="executable" location="extensions">simpletransform.py</dependency>
+  <dependency type="executable" location="extensions">simplestyle.py</dependency>
+  
   <param name="length" type="float" min="0.1" max="100000.0" precision="2" _gui-text="Length of scaling path: " _gui-description="Real-world length of the scaling path">100.0</param>
-  <param name="unit" type="enum" _gui-text="Unit: " _gui-description="Real-world unit for drawing">
-    <item value="px">px</item>
-    <item value="pt">pt</item>
-    <item value="in">in</item>
-    <item value="ft">ft</item>
-    <item value="yd">yd</item>
-    <item value="mm">mm</item>
-    <item value="cm">cm</item>
-    <item value="m">m</item>
-    <item value="km">km</item>
+  <param name="unit" type="optiongroup" appearance="minimal" _gui-text="Unit: " _gui-description="Real-world unit for drawing">
+    <option value="px">px</option>
+    <option value="pt">pt</option>
+    <option value="in">in</option>
+    <option value="ft">ft</option>
+    <option value="yd">yd</option>
+    <option value="mm">mm</option>
+    <option value="cm">cm</option>
+    <option value="m">m</option>
+    <option value="km">km</option>
+  </param>
+  <param name="desc_unit" type="description">Large objects need to be scaled to fit onto the page.</param>
+  <param name="desc_unit" type="description">Select a scale by choosing which list to pick from and then</param>
+  <param name="desc_unit" type="description">selecting a Scale. Or choose Manual, and enter directly.</param>
+  <param name="choosescale" _gui-text="Choose Scale category" type="optiongroup" appearance="minimal">
+    <option value="scales">All Scales</option>
+    <option value="metric">Metric</option>
+    <option value="imperial">Imperial</option>
+    <option value="manual">Manual</option></param>
+  <param name="scales" type="enum" _gui-text="All Scales:              "  _gui-description="All Standard scales for Drawings">
```

Comment:
the non-preferred ones are actually metric but not widely used. Probably we could drop them as we allow manual entry.
I think you're suggesting I drop the all and just have: metric, imperial, manual ??